src/agiletest_cli/agiletest_client.py

```diff
@@ -67,19 +74,27 @@ def update_token(self, response: Response) -> None:
         self.token = str(response.text).strip()
 
     def auth_flow(self, request: httpx.Request) -> Generator[Request, Response, None]:
-        if not self._check_valid_token():
-            self.logger.debug("Refreshing token")
-            refresh_res = yield self.build_refresh_request()
-            self.update_token(refresh_res)
+        if not self.data_center:
```

Comment:
nitpick: should use `if self.data_center`, then you can parse the cloud mode in `else`